probe-rs/src/flashing/loader.rs

```diff
@@ -242,20 +248,13 @@ impl FlashLoader {
         Ok(())
     }
 
-    fn commit_nvm(
+    fn get_flash_algorithm(
```

Comment:
I think idiomatic Rust would be not to name it get_?

Reply:
What do you suggest? Something like `assemble_flash_algorithm`?

Reply:
I guess. Or just 'flash_algorithm'. I don't mind it with get either. It just stuck out to me :)